src/Main/Main.js

```diff
@@ -1,6 +1,8 @@
-import React from 'react'
-import backgroundImg from '../background.png'
-import './Main.css'
+import React from 'react';
+import backgroundImg from '../background.png';
+import './Main.css';
+import { zodiac } from '../data.js';
```

Comment:
There is no need to import this again, because you passed this array in as a prop from the App component.